Neos.Neos/Configuration/Policy.yaml

```diff
@@ -258,6 +258,9 @@ roles:
 
   'Neos.Neos:UserManager':
     privileges:
+      -
+        privilegeTarget: 'Neos.Neos:Backend.GeneralAccess'
```

Comment:
Ah, let me guess: That makes login work, even if *only* `UserManager` as assigned… 👍

Reply:
Yeah, with the current implementation of the backend login and redirect you will end up in the content module where you don't have any access. But if we fix that (or if you know the correct url) you can now access the user module